language/move-lang/src/parser/ast.rs

```diff
@@ -755,6 +762,16 @@ impl fmt::Display for BinOp_ {
     }
 }
 
+impl fmt::Display for FunctionVisibility {
+    fn fmt(&self, f: &mut fmt::Formatter) -> std::fmt::Result {
+        match &self {
+            FunctionVisibility::Public(_) => write!(f, "{}", FunctionVisibility::PUBLIC),
+            FunctionVisibility::Script(_) => write!(f, "{}", FunctionVisibility::SCRIPT),
+            FunctionVisibility::Internal => write!(f, "{}", FunctionVisibility::INTERNAL),
+        }
```

Comment:
If you want you can move it to, `write!(f, "{}", match ....)` not sure if it is better or not